.github/workflows/build_librealsense_ROS2_Ubuntu.yaml

```diff
@@ -0,0 +1,60 @@
+name: Build_ROS2
+
+on:
+  push:
+    branches: ['**']
+  pull_request:
+    branches: ['**']
+
+jobs:
+
+  test_docker:
```

Comment:
Lets change this name to a more meaningful name:
![image](https://user-images.githubusercontent.com/64067618/202121647-f4f884c5-c9b1-4818-82f4-970ccb95cfa8.png)

something like:
`build_lrs_ros2_package`
or
`librealsense2_ros2_package`

Reply:
The job was renamed.